.github/workflows/package-tests.yml

```diff
@@ -63,21 +63,26 @@ jobs:
               DIR=$(_jq '.directory')
               NAME=$(_jq '.name')
               echo "::group::$NAME"
+              TYPE=$(_jq '.type')
               localExit=0
 
-              _file_exist $DIR/.gitattributes || localExit=1
+              if [ $TYPE != 'contract' ] && [ $TYPE != 'contracts' ]; then
+                _file_exist $DIR/.gitattributes || localExit=1
+              fi
               _file_exist $DIR/.gitignore || localExit=1
               _file_exist $DIR/CHANGELOG.md || localExit=1
               _file_exist $DIR/LICENSE || localExit=1
-              _file_exist $DIR/phpunit.xml.dist || localExit=1
+              if [ $TYPE != 'contract' ]; then
+                _file_exist $DIR/phpunit.xml.dist || localExit=1
+              fi
               _file_exist $DIR/README.md || localExit=1
               _file_not_exist $DIR/phpunit.xml || localExit=1
 
               if [ $(_jq '.new') == true ]; then
                 echo "Verifying new package"
                 _correct_license_file $DIR/LICENSE || localExit=1
 
-                if [ $(_jq '.component_bridge') == false ]; then
+                if [ $TYPE == 'component_bridge' ]; then
                   if [ ! $(cat composer.json | jq -e ".replace.\"$NAME\"|test(\"self.version\")") ]; then
                     echo "Composer.json's replace section needs to contain $NAME"
```

Comment:
why is this done only for component bridges ? bundles, normal bridges (other than the phpunit one which is special) and components (other than runtime which is a composer plugin) should also be replaced.

And `contract` type should be replaced in `src/Symfony/Contracts/composer.json` file